### PR TITLE
Add org.freedesktop.fldigi

### PR DESCRIPTION
**Pull request by lu9dce**
my first flatpak

Fldigi is a free and open-source program which allows an ordinary computer's sound card to be used as a simple two-way data modem. The software is mostly used by amateur radio operators who connect the microphone and headphone connections of an amateur radio SSB transceiver or an FM two way radio to the computer's headphone and microphone connections, respectively. This interconnection creates a "sound card defined radio" whose available bandwidth is limited by the sound card's sample rate and the external radio's bandwidth. Such communications are normally done on the shortwave amateur radio bands in modes such as PSK31, MFSK, RTTY, Olivia, and CW. Increasingly, the software is also being used for data on VHF and UHF frequencies using faster modes such as 8-PSK. Using this software, it is possible for amateur radio operators to communicate worldwide while using only a few watts of RF power

### CODE REVIEW

File: org.freedesktop.fldigi.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+app-id: org.freedesktop.fldigi
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: fldigi
+finish-args:
+  - --share=ipc
+  - --fallback-x11
```
Comment (bilelmoussaoui) [inline org.freedesktop.fldigi.yml:8]:
```suggestion
  - --socket=fallback-x11
```

**Comment by LeandroStanger** [comment]
bot, build org.freedesktop.fldigi

**Comment by lu9dce** [comment]
what is this how i fix it?

stat: cannot stat 'builddir/*/share/app-info/xmls/org.freedesktop.fldigi.xml.gz': No such file or directory

File: org.freedesktop.fldigi.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>org.freedesktop.fldigi</id>
```
Comment (nedrichards) [inline org.freedesktop.fldigi.metainfo.xml:4]: is this really the app-id? e.g. is this actually part of the freedesktop.org project?

File: org.freedesktop.fldigi.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>Fldigi</name>
+ <summary>Amateur Radio Sound Card Communications</summary>
+ <developer_name>dave-w1hkj, mdblack98, sb42</developer_name>
+ <url type="homepage">https://sourceforge.net/projects/fldigi/
```
Comment (nedrichards) [inline org.freedesktop.fldigi.metainfo.xml:10]: this tag isn't closed so the xml is invalid and it fails to compile properly.

File: flathub.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (nedrichards) [inline flathub.json:2]: what about this doesn't work on aarch64?

Comment (lu9dce) [reply flathub.json:2]: I have no way to ensure that

File: org.freedesktop.fldigi.metainfo.xml (hunk 3 above)
Comment (lu9dce) [reply org.freedesktop.fldigi.metainfo.xml]: ok

File: org.freedesktop.fldigi.metainfo.xml (hunk 2 above)
Comment (lu9dce) [reply org.freedesktop.fldigi.metainfo.xml:4]: I really can't answer this to you .. where did I get that information from? or what corresponds to fence?

**Comment by lu9dce** [comment]
How often does the bot control?

**Pull request closed without merging**